data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/IdsMultipartApiServiceExtension.java

```diff
@@ -74,58 +75,32 @@ public final class IdsMultipartApiServiceExtension implements ServiceExtension {
     public static final String EDC_IDS_ID = "edc.ids.id";
     public static final String DEFAULT_EDC_IDS_ID = "urn:connector:edc";
 
-    private Monitor monitor;
-    @Inject
-    private WebService webService;
-    @Inject
-    private IdentityService identityService;
-    @Inject
-    private CatalogService dataCatalogService;
-    @Inject
-    private ConnectorService connectorService;
-    @Inject
-    private AssetIndex assetIndex;
-    @Inject
-    private IdsTransformerRegistry transformerRegistry;
-    @Inject
-    private ContractOfferService contractOfferService;
-    @Inject
-    private ContractNegotiationStore contractNegotiationStore;
-    @Inject
-    private ConsumerContractNegotiationManager consumerNegotiationManager;
-    @Inject
-    private ProviderContractNegotiationManager providerNegotiationManager;
-    @Inject
-    private TransferProcessManager transferProcessManager;
-    @Inject
-    private ContractValidationService contractValidationService;
-    @Inject
-    private EndpointDataReferenceReceiverRegistry endpointDataReferenceReceiverRegistry;
-    @Inject
-    private EndpointDataReferenceTransformerRegistry endpointDataReferenceTransformerRegistry;
-    @Inject
-    private IdsApiConfiguration idsApiConfiguration;
-    @Inject
-    private ObjectMapperFactory objectMapperFactory;
-    @Inject
-    private Vault vault;
+    @Inject private Monitor monitor;
```

Comment:
I like this way of formatting annotations, but I don't know if it's good to have to ways of formatting them

Reply:
What do the others think?